etc/terminal-run

```diff
@@ -21,44 +21,83 @@ function error {
   exit 1
 }
 
-if [ -f /usr/bin/lxterminal ];then
+# Check if the current terminal is in /usr/bin - Wallee
+if ! [ -f /usr/bin/$(cat ~/pi-apps/data/settings/Terminal) ];then
+
+  if [ -f /usr/bin/lxterminal ];then
```

Comment:
```suggestion
  if command -v lxterminal &>/dev/null; then
```

This will check to see if the command exists, regardless of the location, as long as it is on path.
It would probably be a good idea to use this for all the `elif`s too.